lib/google/container/property/cluster_horizontal_pod_autoscaling.rb

```diff
@@ -80,17 +80,15 @@ def compare_fields(other)
       # Data is coming from the GCP API
       class ClustHorizPodAutosApi < ClustHorizPodAutos
         def initialize(args)
-          @disabled =
-            Google::Container::Property::Boolean.api_munge(args['disabled'])
+          @disabled = Google::Container::Property::Boolean.api_munge(args['disabled'])
         end
       end
 
       # Manages a ClustHorizPodAutos nested object
```

Comment:
Can the comments use the real full name of the object instead of the autogenerated short one?